docs/src/pages/zh-TW/getting-started.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: 入門指南
```

Comment:
```suggestion
title: 入門指南
lang: zh-TW
```
Could you add this so that the lang attribute gets set on the html element?

Reply:
Someone else is responsible for the translation zh-TW ，So it's not added lang: zh-TW ，Wait for @FredKSchott reply